platform/src/main/java/org/hillview/utils/Utilities.java

```diff
@@ -54,11 +54,6 @@ public static String wildcardToRegex(@Nullable String wildcard) {
                 case '*':
                     s.append(".*");
                     break;
-                case '?':
```

Comment:
Why are you deleting this code?
The user types a *shell* pattern, and the Java code runs a regular expression. This code converts a shell pattern into a regular expression.

Reply:
@mbudiu-vmw The shell pattern is interpreted as regular expression so conversion was not required. 

![dot](https://user-images.githubusercontent.com/9160770/48244569-747eaf00-e40c-11e8-975e-ef92d448bec1.PNG)
![question_mark](https://user-images.githubusercontent.com/9160770/48244622-aee84c00-e40c-11e8-8f78-52d885386166.PNG)

Reply:
it is a different regular expression.
In the shell ? means the same as dot: one arbitrary character.
As a regex it is a modifier meaning: the previous regex is optional.